### PR TITLE
Add ingress to gitlab-ce chart

### PR DESCRIPTION
**Pull request by goshlanguage**
Added an ingress template based on what I see for Jenkins.

Please let me know if further documentation is needed or if there are any style suggestions or other possible improvements.

### CODE REVIEW

File: stable/gitlab-ce/templates/ingress.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{{- if .Values.ingress.enabled }}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+{{- if .Values.ingress.annotations }}
+  annotations:
+{{ toYaml .Values.ingress.annotations | indent 4 }}
+{{- end }}
+  name: {{ template "fullname" . }}
```
Comment (unguiculus) [inline stable/gitlab-ce/templates/ingress.yaml:9]: Add standard labels.

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/gitlab-ce/templates/ingress.yaml (hunk 1 above)
Comment (goshlanguage) [reply stable/gitlab-ce/templates/ingress.yaml:9]: I've added labels below this, thanks for the feedback

Comment (goshlanguage) [reply stable/gitlab-ce/templates/ingress.yaml:9]: @unguiculus does this cover standard labels appropriately?

**Comment by unguiculus** [comment]
@k8s-bot ok to test

**Review by unguiculus — APPROVED** [review]

File: stable/gitlab-ce/templates/ingress.yaml (hunk 1 above)
Comment (unguiculus) [reply stable/gitlab-ce/templates/ingress.yaml:9]: Sorry for the late feedback. With respect to the existing chart, it does. For new charts, the recommendation is to use `{{ template "name" . }}` for the `app` label.

**Pull request merged**